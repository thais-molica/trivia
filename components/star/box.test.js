import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Star from "./index";

it("Star renders correctly", () => {
  const tree = renderer.create(<Star alt="alt" />).toJSON();
  expect(tree).toMatchSnapshot();
});
