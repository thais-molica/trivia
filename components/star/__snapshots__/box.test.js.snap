// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Star renders correctly 1`] = `
<img
  alt="alt"
  className="jsx-1677436475"
  src={[Function]}
/>
`;
